### DISCUSSION TITLE
Fails with unknown error on captcha during initial install

### DISCUSSION THREAD

**Discussion by joeinaustin** [fuatakgun/eufy_security#727] [General]
Is anyone else unable to get past the captcha during install?  I have entered the captcha many times.  The log says my credentials are not "activated" but they work with the actual eufy app.    I have been unable to get past this error.

**Comment by omar10wahab** [comment]
i am now also having this problem. I was having the add on crashing for awhile until i noticed someone said they disabled all their addons and ran the Eufy one and it worked. But now when i tried reinstalling everything the captcha shows up in the devices and it keeps giving me errors.

**Reply by omar10wahab** [reply]
Nevermind, i restarted my homeassisstant and immediately after boot up entered the captcha and it worked
